### ISSUE TITLE
Long delay on lighting updates

### ISSUE THREAD

**Issue by bloxgate** [Electrical-Age/ElectricalAge#474]
Encountered this while putting some lights in a new building at my base. After turning on the lights it takes several minutes for the lights to update and actually start shining on the ground and walls. Then once they're turned off some of the lights take a few minutes to actually update to off.

Lights turned on:
![2016-07-02_19 55 41](https://cloud.githubusercontent.com/assets/2079324/16625232/206bd5fa-4369-11e6-8adc-938b6d035d20.png)

Lights turned off:
![2016-07-04_01 15 52](https://cloud.githubusercontent.com/assets/2079324/16625238/2985cc40-4369-11e6-9652-66aae642f0bd.png)

**Comment by Dolu1990** [comment]
Hi Bloxgate,
Is that with the very last version ? (A big improvement was done recently, it's just to be sure of this issue context.)

**Comment by bloxgate** [comment]
Yes I am using the latest release. I just did some testing with Kim_Bruning and it seems that breaking and replacing one of the lights causes them to all update once I got within break range of them.

**Comment by Baughn** [comment]
This is an end-of-life server, however. It might be due to throttling of lighting updates somewhere?

EDIT: Nm, different server.

**Comment by bloxgate** [comment]
That's not on E12...

**Comment by cm0x4D** [comment]
Whats the environment? 
Which other mods are running?
Any custom shaders?

Could we have a copy of the world or can the bug easily be reproduced?

**Comment by bloxgate** [comment]
I'm not sure how to reproduce as I haven't been able to get it to occur when testing lights by themselves, however I have seen it occasionally on other servers. World can be downloaded from [here](http://beefyserv.bloxgaming.com/bloxgate/world.tar.gz)

Don't have any custom shaders installed.

**Client Environment**
Forge: 1.7.10-1614
Windows 10 64-bit
Java 8u91 64-bit
Mods:
- AgriCraft-1.7.10-1.5.0.jar
- AOBD-2.9.2.jar
- appliedenergistics2-rv2-stable-10.jar
- archimedesshipsplus-1.7.10-1.8.1.jar
- bdlib-1.9.4.109-mc1.7.10.jar
- BiblioCraft[v1.11.5][MC1.7.10].jar
- BiblioWoods[Natura][v1.5].jar
- ChromatiCraft 1.7.10 V15a.jar
- CodeChickenCore-1.7.10-1.0.7.47-universal.jar
- DragonAPI 1.7.10 V15a.jar
- dynamictanks2-1.1.05.jar
- ElectricalAge_BETA-1.11_r51.jar
- ElectriCraft 1.7.10 V15a.jar
- EnderCore-1.7.10-0.2.0.36_beta.jar
- EnderIO-1.7.10-2.3.0.427_beta.jar
- EnderStorage-1.7.10-1.4.7.38-universal.jar
- extrautilities-1.2.12.jar
- ImmersiveEngineering-0.7.7.jar
- industrialcraft-2-2.2.822-experimental.jar
- ironchest-6.0.62.742.jar
- Jabba-1.2.2_1.7.10.jar
- journeymap-1.7.10-5.1.4p1-unlimited.jar
- Mantle-1.7.10-0.3.2.jar
- MatterOverdrive-1.7.10-0.4.1.jar
- Mekanism-1.7.10-9.0.2.271.jar
- MekanismGenerators-1.7.10-9.0.2.271.jar
- MekanismTools-1.7.10-9.0.2.271.jar
- movingworld-1.7.10-1.8.1.jar
- natura-1.7.10-2.2.0.1.jar
- neiaddons-1.12.13.38-mc1.7.10.jar
- NEIIntegration-MC1.7.10-1.1.2.jar
- NotEnoughItems-1.7.10-1.0.5.120-universal.jar
- OpenBlocks-1.7.10-1.5.1.jar
- OpenModsLib-1.7.10-0.9.1.jar
- OptiFine_1.7.10_HD_U_D6.jar
- pressure-1.3.0.131-mc1.7.10.jar
- ProjectE-1.7.10-PE1.10.0.jar
- ReactorCraft 1.7.10 V15a.jar
- RotaryCraft 1.7.10 V15a.jar
- Somnia-1.4.8.55.jar
- tcnodetracker-1.7.10-1.1.2.jar
- TConstruct-1.7.10-1.8.8.jar
- Thaumcraft-1.7.10-4.2.3.5.jar
- thaumcraftneiplugin-1.7.10-1.7a.jar
- ThaumicEquivalence-1.7.10-1.0.3.jar
- ThaumicExploration-1.7.10-1.1-53.jar
- ThaumicTinkerer-2.5-1.7.10-164.jar
- Waila-1.5.10_1.7.10.jar
- Wawla-1.0.5.120.jar

**Server**
Ubuntu 14.04 LTS
Same mods and same java version
Running Thermos-1.7.10-1614

**Comment by bowserjr2424** [comment]
Do you think I could join your server @bloxgate

**Comment by AMIDIBOSS** [comment]
OptiFine_1.7.10_HD_U_D6.jar causes the issue, i had the same. Enable dynamic lights - that fixes the problem

**Comment by bloxgate** [comment]
Thanks I'll give that a try.

**Comment by bloxgate** [comment]
Nope, issue occurs even without Optifine.

**Comment by AMIDIBOSS** [comment]
I don't know, tray random graphics configuration. eneble or disable whathever is in there.
And remember your'e playing "Minecraft, the order of Bugs", that happens even with standard torches.

**Comment by bloxgate** [comment]
@cm0x4D Is there any other info you need? I noticed that the needs more info label is still there.
